api/v1alpha1/flowcollector_webhook.go

```diff
@@ -59,9 +59,27 @@ func (r *FlowCollector) ConvertTo(dstRaw conversion.Hub) error {
 
 	dst.Spec.Loki.StatusTLS = restored.Spec.Loki.StatusTLS
 
+	if restored.Spec.Exporters != nil {
+		for _, restoredExp := range restored.Spec.Exporters {
+			if !isExporterIn(restoredExp, dst.Spec.Exporters) {
+				dst.Spec.Exporters = append(dst.Spec.Exporters, restoredExp)
+			}
+		}
+	}
+
 	return nil
 }
 
+func isExporterIn(restoredExporter *v1beta1.FlowCollectorExporter, dstExporters []*v1beta1.FlowCollectorExporter) bool {
+
+	for _, dstExp := range dstExporters {
+		if restoredExporter == dstExp {
```

Comment:
Shouldn't we use [reflect.DeepEqual](https://pkg.go.dev/reflect#DeepEqual) here ? :thinking: